packages/commerce-sdk-react/src/test-utils.tsx

```diff
@@ -5,36 +5,33 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import {render, RenderOptions} from '@testing-library/react'
-import jwt from 'jsonwebtoken'
-import nock from 'nock'
 import React from 'react'
-import CommerceApiProvider, {CommerceApiProviderProps} from './provider'
+import {render, RenderOptions} from '@testing-library/react'
+import { renderHook } from '@testing-library/react-hooks/dom'
 import {QueryClient, QueryClientProvider} from '@tanstack/react-query'
-import {url} from 'inspector'
+import CommerceApiProvider, {CommerceApiProviderProps} from './provider'
 
-export const TEST_CONFIG = {
-    proxy: 'http://localhost:3000/mobify/proxy/api',
-    clientId: 'c9c45bfd-0ed3-4aa2-9971-40f88962b836',
-    organizationId: 'f_ecom_zzrf_001',
-    shortCode: '8o7m175y',
-    redirectURI: 'http://localhost:3000/callback',
+export const DEFAULT_TEST_CONFIG = {
+    // localhost:8888 does NOT exist
+    // it is intentional b/c we want to catch the
+    // requests that are not intercepted by nock
+    proxy: 'http://localhost:8888/mobify/proxy/api',
+    redirectURI: 'http://localhost:8888/callback',
+    clientId: '12345678-1234-1234-1234-123412341234',
+    organizationId: 'f_ecom_zzrmy_orgf_001',
+    shortCode: '12345678',
```

Comment:
So for the auth flow we're hitting `localhost:8888` which is **not** intercepted (per your code comment here), does that mean all instances of `localhost:3000` are replaced with `localhost:8888` and then I see you `jest.mock`ing `Auth` inside `CommerceApiProvider`, am I following that correctly?

Reply:
that's correct. All instances of `localhost:3000` is replaced.